Automattic/sensei
# Add some samples as Sensei patterns

Fixes #5137

### Changes proposed in this Pull Request

* It registers some samples as Sensei patterns (copied from the patterns directory). There are 2 methods to register patterns for courses or lessons.
* It also registers the Sensei category.

### Testing instructions


* Create a new course, navigate through the wizard, and make sure you see the registered patterns in the last step.
* Also, check that you can see the patterns in the inserter filtering it by "Sensei LMS" category.
* Repeat the test with a new lesson.

### Known issues

* Sometimes, I see some weird behaviors with the `BlockPreview` component, displaying it with the wrong height. If I activate the Gutenberg plugin, it works better. I think we could postpone our decision of what to do, at the moment that we create the final patterns to see if the issue will happen with them. If yes, I think we could set a maximum height to the thumbnails as a temporary workaround until the Gutenberg fix is in the versions we support.


### Screenshot / Video

<img width="1488" alt="Screen Shot 2022-05-23 at 19 36 23" src="https://user-images.githubusercontent.com/876340/169915743-06ffadc0-df4e-417d-96ed-197dd5391e89.png">

<img width="402" alt="Screen Shot 2022-05-23 at 19 36 47" src="https://user-images.githubusercontent.com/876340/169915751-f82ec802-92b8-4d69-9218-b5841486d7b4.png">
